src/locales/en.json

```diff
@@ -277,7 +278,15 @@
       "bars": "Went to a bar in the last 10 days",
       "microcovid_budget_one_percent": "Keeps to a budget of 200 microCOVID per week (\"Standard Caution Budget\")",
       "microcovid_budget_deci_percent": "Keeps to a budget of 20 microCOVID per week (\"High Caution Budget\")",
-      "hasCovid": "Has COVID"
+      "hasCovid": "Has COVID",
+      "notSure_short": "Not sure",
+      "hasCovidRightNow_short": "Infected with COVID",
+      "minimalExposure_short": "1-2 person household, minimal exposure",
+      "minimalExposure": "Once a week groceries; all socializing masked, outdoors and distanced",
```

Comment:
I mildly want to capitalize AND to emphasize that they are doing all three.

Also, can we get an oxford comma on that list? :)